### ISSUE TITLE
About Validation Set？

### ISSUE THREAD

**Issue by RRRRendy** [Wayfear/BrainNetworkTransformer#7]
您的代码中对数据集进行了训练测试验证的划分，想请教您通常这个过程是训练集上训练，调整参数使得验证集上表现最好，然后进行测试集的测试？
还是训练集上训练，测试集上测试，拿到测试集上表现最好的模型，参数保存下来，用这个模型去测试集上测试？

希望能得到您的解答，感激不尽

**Comment by Wayfear** [comment]
We do it as we mentioned in our paper, "All models are trained for 200 epochs, and the epoch with
the highest AUROC performance on the validation set is used for performance comparison on the
test set."
